src/components/HeaderItems/LogoutButton.jsx

```diff
@@ -0,0 +1,31 @@
+import { useNavigate } from "react-router-dom";
+import { useMutation } from "@tanstack/react-query";
+
+import fetchData from "../../utils/axios";
+
+import Button from "../shared/Button";
+
+function LogoutButton() {
+  const navigate = useNavigate();
+
+  async function handleGoogleLogout() {
+    await fetchData("POST", "/auth/logout");
+  }
+
+  const { mutate } = useMutation(handleGoogleLogout, {
+    onSuccess: () => {
+      navigate("/login");
+    },
+    onFailure: () => {
+      console.log("sending user to errorpage");
+    },
```

Comment:
onSuccess와 onFailure을 어제 리뷰받은 내용을 바탕으로 잘 작성해주셨네요!

저희 코드 컨벤션에 맞게 화살표 함수를 함수 선언식으로 수정하는걸 제안드려봅니다만, 제가 아직 React Query 신택스에 대해 부족한 부분이 있어, 한번 검토해보시면 좋겠습니다.

```suggestion
    onSuccess: function() {
      navigate("/login");
    },
    onFailure: function() {
      console.log("sending user to errorpage");
    },
```